CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 2.28.0 (2021-04-22)
+
+### Breaking change
+- **Translate**: [ebrehault]
+    - Translations are now added from any module using the `addTranslations` static method
+    - `mergeTranslations` is not available anymore
```

Comment:
Technically it still is.  If you don't want it to be, make it a non-exported constant in the module.

Reply:
oh well let me remove it then, I do not see any use case where we would need this function outside the module